portfolio/src/main/webapp/script.js

```diff
@@ -13,19 +13,30 @@
 // limitations under the License.
 
 async function getPhrase(){
-    let response = await fetch('/load-data');
-    let list = await response.json();
-    if(list.length != 0){
-        for(i = 0; i < list.length; i++){
-            let commentSection = document.createElement("DIV");
-            commentSection.setAttribute("id", "dynamic-history");
-            let addName = document.createTextNode(list[i].name + ": ");
-            let addMessage = document.createTextNode(list[i].message);
-            commentSection.appendChild(addName);  
-            commentSection.appendChild(addMessage);  
-            document.getElementById("section").appendChild(commentSection);
-        }
+    let max = document.getElementById("max-comments").value;
+    let response = await fetch('/load-data?numCommentsDisplay=' + max);
+    let list = await response.json();//list of entities from datastore
+    let commentHistory = document.getElementById("section");//UI for displaying comments
+
+    while (commentHistory.hasChildNodes()) {
+        commentHistory.removeChild(commentHistory.childNodes[0]);
+    }
```

Comment:
Looks like there is an easier solution to this:
https://stackoverflow.com/questions/3955229/remove-all-child-elements-of-a-dom-node-in-javascript

Reply:
I tried clearing the inner HTML but since I was using div children inside of the commentHistory div, I was unable to clear the child elements. I used an option similar to 2A and 2B

Reply:
I'll send a commit with a better implementation. I was able to set the innerHTML to nothing to clear out the comments